HACKING.adoc

```diff
@@ -261,6 +261,10 @@ Then run:
  $ gem push asciidoctor-revealjs-X.Y.Z.gem
 
 . Check that the new version is available on https://rubygems.org/gems/asciidoctor-revealjs[rubygems.org]
+. Generate a compatible version of the Ruby converter (using opal mode)
++
+ $ bundle exec rake build:converter:opal
```

Comment:
A `rake clean` is missing here but I think it should be done automatically whenever we build a template converter. Can you think of a reason why we shouldn't?

Reply:
I agree, the build task should always generate the file (even if it already exists) since we can generate the file using different modes.